features/step_definitions/transaction_steps.rb

```diff
@@ -22,3 +22,8 @@
 When /^the buyer makes (\d+|a) service transactions? with:$/ do |count, table|
   step %(buyer "#{@buyer.name}" makes #{count} service transactions with:), table
 end
+
+Given "the backend will respond to a utilization request for the application with:" do |json|
```

Comment:
I think I addressed all your comments. Now I have a question, I tried defining this step like this:

```suggestion
Given "the backend will respond to a utilization request for {application} with:" do |application, json|
```
To follow the same formula as in [here](https://github.com/3scale/porta/pull/3558/files#diff-235c89ea8624bdbdda33252cc58951529ef4bc59a43e7b079df8b7b1a2425008R69).

But it failed to take the application correctly. Do you know why it works for the buyer case above but not for this case with the application?